packages/remix-server-runtime/magicExports/remix.ts

```diff
@@ -19,13 +19,17 @@ export type {
   HtmlLinkDescriptor,
   PageLinkDescriptor,
   ErrorBoundaryComponent,
+  ErrorBoundaryProps,
```

Comment:
@mbrowne I suppose you want these to be available from `@remix-run/cloudflare` & `@remix-run/node` too?
We're moving away from using `magicExports` & `@remix-run/server-runtime` isn't seen as a public API (see #2359)

Reply:
No problem and thanks for the link. I added the exports to `@remix-run/cloudflare` and `@remix-run/node`. Let me know if I should also remove them from `remix-server-runtime/magicExports/remix.ts` since those magic exports will be going away at some point.

Reply:
We currently keep the `magicExports` to be backwards compatible.
We'll remove them in v2 though.